server/views/serviceProviderReferrals/endOfServiceReport/formTemplate.njk

```diff
@@ -13,10 +13,6 @@
 
       <h1 class="govuk-heading-xl">{{ presenter.formPagePresenter.text.title }}</h1>
 
-      {% if presenter.formPagePresenter.text.pageNumber %}
```

Comment:
Was removing the page numbers from end of service report an accident?

Reply:
Yep, have fixed this.